src/contrib/coding-style.rst

```diff
@@ -1,12 +1,494 @@
 Coding Style
+############
+
+Introduction
 ============
+This manual contains coding standards and guidelines for SSSD contributors.
+All new code should adhere to these standards. Those rules does not apply to
+existing legacy code. Please do not go back and apply formatting changes to
+the old code. It just confuses things and is generally a waste of time.
+
+Why we are using coding guidelines?
+-----------------------------------
+The Coding Guidelines are necessary to improve maintainability and readability
+of the code. It allows contributors from all over the world to stick to the one
+code convention making code colaboration much easier.
+
+Improved readability
+--------------------
+Code is not only for compilers, is mainly for peoples too.
```

Comment:
s/is mainly for peoples too/it is for people also/

Reply:
Fixed